src/upload.ts

```diff
@@ -246,7 +246,7 @@ export async function uploadLargeFileRequest(
   });
 
   let parallelUploads = 1;
-  if (resp.headers["Tus-Extension"]?.contains("concatenation")) {
+  if (resp.headers["tus-extension"]?.includes("concatenation")) {
```

Comment:
I think there **should** be a `get` method on `Headers` object that you should use instead of accessing the header in an array-like way. There should be something like `resp.headers.get("tus-extension")` that is case insensitive and normalises header name. Could you check whether something like this is available?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Headers

Reply:
Sounds good to me, but I'm getting

```
getJSON › should perform a lookup and skylink GET

    File content response invalid despite a successful request. Please try again and report this issue to the devs if it persists. TypeError: response.headers.get is not a function

      654 |     validateRegistryProofResponse(inputSkylink, skylink, proof);
      655 |   } catch (err) {
    > 656 |     throw new Error(
          |           ^
      657 |       `File content response invalid despite a successful request. Please try again and report this issue to the devs if it persists. ${err}`
      658 |     );
      659 |   }

      at validateGetFileContentResponse (download.ts:656:11)
      at SkynetClient.<anonymous> (download.ts:398:3)
      at step (download.ts:8879:19)
      at Object.next (download.ts:8609:14)
      at fulfilled (download.ts:8496:24)
```

1000+ of these errors at runtime. Any idea why?

Every search I make for this tells me to use the array indexing notation for axios e.g. https://github.com/axios/axios/issues/1324#issuecomment-442717563.

Reply:
yeah I think axios uses a different response type and it's just plain key-value object so we don't get those function utils unfortunately, we should probably just go with what you had before

reference https://github.com/axios/axios/blob/master/index.d.ts#L88